plugins/obs-filters/data/locale/en-US.ini

```diff
@@ -35,6 +35,10 @@ KeyColor="Key Color"
 Similarity="Similarity (1-1000)"
 Smoothness="Smoothness (1-1000)"
 ColorSpillReduction="Key Color Spill Reduction (1-1000)"
+ColorSpillReductionType="Key Color Spill Reduction Type"
+SPILL_TYPE_SATURATION="Reduce saturation"
+SPILL_TYPE_GREEN_AVG="Limit Green by avg(red,blue)"
+SPILL_TYPE_BLUE_AVG="Limit Blue by avg(red,green)"
```

Comment:
I think "Limit Green" and "Limit Blue" would be sufficient here. Most users don't need to be aware of the mathematical operation being performed, only what the effect of a selection will be.